liquidator/detail/score_matrix_detail.h

```diff
@@ -79,10 +79,10 @@ scale(const PWM& pwm, const std::pair<double, double>& min_max, const unsigned r
 }
 
 // returns score; sequences with invalid characters return 0.
-unsigned score(const std::vector<std::array<unsigned, AlphabetSize>>& matrix,
-               const std::string& sequence,
-               const size_t begin,
-               const size_t end)
+inline unsigned score(const std::vector<std::array<unsigned, AlphabetSize>>& matrix,
```

Comment:
Note that I inline all this stuff just so I don't get duplicate definition linker errors.  (Maybe I'm just lazy; wherever I can I like to avoid having a declaration separate from implementation.)  I trust compiler to inline what it sees fit regardless of what I declare inline.